test/api.js

```diff
@@ -1,17 +1,44 @@
 var request = require('supertest');
 
-// provide a session secret for testing
-var config = {
-  SESSION_SECRET: 'test_secret'
-};
+// test config (need to override .env and .env.dist)
```

Comment:
What badgekit are we planning to test on? Not http://badgekit-api-sciencelab.herokuapp.com/? Ohh, were you asking me to setup a separate badgekit-api with "golden" data? Sorry I think I missed all that.

Can we test against our current badgekit server? We can use the defaults in env.dist (except BADGES_SECRET, I guess)